### PR TITLE
Open ranger at the path of the current file, maintain working directory

### PR DESCRIPTION
**Pull request by benwoodward**
What this does is open ranger at the path of the current file / directory from vim's perspective. It also respects the vim-user's working directory setting (`s:original_dir`), which can be set by the .vimrc, e.g.

`autocmd VimEnter * silent! lcd %:p:h " Set working directory to current file / dir from perspective of vim`

`autocmd VimEnter * silent! lcd $PWD " Set working directory to current file / dir from perspective of shell`

### CODE REVIEW

File: autoload/floaterm/wrapper/ranger.vim [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 function! floaterm#wrapper#ranger#() abort
   let s:ranger_tmpfile = tempname()
-  let cmd = 'ranger ' . $PWD . ' --choosefiles=' . s:ranger_tmpfile
+  let s:original_dir = getcwd()
```
Comment (voldikss) [inline autoload/floaterm/wrapper/ranger.vim]: It's not necessary to use script-level variable name here, just use `original_dir`

Comment (benwoodward) [reply autoload/floaterm/wrapper/ranger.vim]: Pushed a new commit.

**Comment by benwoodward** [comment]
Pushed a commit to fix `fff` wrapper with same solution.

**Comment by voldikss** [comment]
Merged, thank you for the contribution!

**Comment by benwoodward** [comment]
No worries!

**Pull request merged**